src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/JSMarshalerType.cs

```diff
@@ -197,6 +221,8 @@ public static JSMarshalerType Task()
         /// <summary>
         /// Marshal as JavaScript <see href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise">Promise</see>.
         /// </summary>
+        /// <returns>Marshaler metadata</returns>
+        /// <param name="result">Metadata about type of the Task result value</param>
```

Comment:
Suggestion applies to all the similar texts.
This method does not seem to be talking about our System.Threading.Tasks.Task, is it? If that's the case, then I would lowercase "task".
If it's talking about System.Threading.Tasks.Task, I would instead use the full see cref: `<see cref="System.Threading.Tasks.Task"/>`
```suggestion
        /// <param name="result">Metadata about the type of the task result value.</param>
```